components/user_settings/general/user_settings_general.tsx

```diff
@@ -650,7 +697,7 @@ class UserSettingsGeneralTab extends React.PureComponent {
                 />
             );
         } else {
-            let describe = '';
+            let describe: any = '';
```

Comment:
Should be a `string`, how come it's being cast as an `any`?

Reply:
Same with all the ones below.

Reply:
It looks like we're allowing for either an `Element` or a `string` to be rendered.
You can define the type as `Element | string`.